deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -69,16 +55,7 @@ services:
     volumes:
       - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
     ports:
-      - "1777:1777"   # pprof extension
-      - "8887:8888"   # Prometheus metrics exposed by the agent
-      - "14268:14268"       # Jaeger receiver
```

Comment:
@ankitnayan Have we removed support for Jaeger Receiver and all other receivers for which exposed ports are removed?